.github/workflows/deny.yml

```diff
@@ -13,4 +13,5 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
-    - uses: EmbarkStudios/cargo-deny-action@v2
+    - uses: EmbarkStudios/cargo-deny-action@v2.0.5
+      rust-version: "1.81.0"
```

Comment:
Not great because it introduces some duplication between `rust-toolchain.toml` and this YAML, but it should unblock the CI.